src/scripts/main.js

```diff
@@ -1 +1,78 @@
 'use strict';
+
+const body = document.querySelector('body');
+const message = document.createElement('div');
+
+function createPromise() {
+  const resolver = (resolve, reject) => {
+    body.addEventListener('click', () => {
+      resolve('First promise was resolved');
+    });
+
+    setTimeout(() => {
+      reject('First promise was rejected');
+    }, 3000);
+  };
+
+  return new Promise(resolver);
+}
+
+function createPromise2() {
+  const resolver = (resolve, reject) => {
+    body.addEventListener('click', () => {
+      resolve('Second promise was resolved');
+    });
+
+    body.addEventListener('contextmenu', (e) => {
+      e.preventDefault();
+      resolve('Second promise was resolved');
+    });
+  };
+
+  return new Promise(resolver);
+}
+
+function createPromise3(button) {
+  return new Promise(resolve => {
+    body.addEventListener(button, (evt) => {
+      evt.preventDefault();
+      resolve();
+    });
+  });
+}
+
+const promise1 = createPromise();
+
+promise1.then(result => {
+  message.innerHTML = result;
+  message.classList = 'success';
+  body.append(message);
+
+  const promise2 = createPromise2();
+
+  return promise2;
+})
+  .then(result => {
+    message.innerHTML = result;
+    message.classList = 'success';
+    body.append(message);
+
+    const promise31 = createPromise3('click');
+
+    return promise31;
+  })
+  .then(() => {
+    const promise32 = createPromise3('contextmenu');
+
+    return promise32;
+  })
+  .then(() => {
+    message.innerHTML = 'Third promise was resolved';
+    message.classList = 'warning';
+    body.append(message);
+  })
+  .catch(result => {
+    message.innerHTML = result;
+    message.classList = 'warning';
+    body.append(message);
+  });
```

Comment:
All 3 promises should be independed by each other. 

```
promise1.then().catch()
promise2.then().catch()
promise3.then().catch()
```

Also I should see 3 different messages. Something like this
![2023-05-15_14h49_09](https://github.com/mate-academy/js_promises_practice_DOM/assets/61904995/8035cfeb-47e9-4ceb-b708-57169bc05134)
![2023-05-15_14h49_22](https://github.com/mate-academy/js_promises_practice_DOM/assets/61904995/235059a1-73d1-4739-9d31-b0773545af50)